routeviews_google_upload/README.md

```diff
@@ -5,19 +5,31 @@
 This project provides a (Python) client that takes a file and sends it to the Google Cloud 
 Storage solution for [the Google+RouteViews project](https://github.com/routeviews/google-cloud-storage).
 
-## Installation and Usage
+## Install
 
 This solution is deployed to PyPI, so you can install it simply using the command `pip install 
 routeviews-google-upload`.
 
     pip install routeviews-google-upload
 
+## Examples
+
 The simplest invocation of this tool is to upload a file to a target gRPC server.
 
     routeviews-google-upload --dest https://grpc.routeviews.org --file /bgpdata/2021.03/UPDATES/update.20210331.2345.bz2
 
 > Run the command with the `--help` argument to see all the expected and available arguments.
 
+### Google Cloud Storage server, with Authentication
+
+If the targeted gRPC server is backed by a Google Cloud Storage (GCS) instance, it may require authentication.
```

Comment:
I think the actual problem here is that the cloudrun instance(s) require authentication (over https), before accepting the gRPC connection (which also arrives over https).

This is 'rpc over http(s)', and is.. super confusing ;( (to me anyway!)

Reply:
Aha, I'll update the wording to indicate that cloudrun backend requires Auth (and forget I ever mentioned GCS...). Thanks!